packages/data/es-shim-like/src/is-nan.ts

```diff
@@ -0,0 +1,3 @@
+import { defineEsShim } from '@nolyfill/shared';
+
+export default defineEsShim(isNaN);
```

Comment:
```suggestion
export default defineEsShim(isNaN, true);
```

Static method should specify `true` here.